src/test/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipServiceTest.java

```diff
@@ -695,4 +699,26 @@ void shouldIgnoreMissingConditionsOfJoining() throws SchedulerException {
     assertThat("Unexpected CoJ synced at.", jobDataMap.get(COJ_SYNCED_FIELD),
         is(nullValue()));
   }
+
+  @Test
+  void shouldIncludeDeferralInAppNotification() throws SchedulerException {
```

Comment:
This is already covered under the parameterised test `shouldAddInAppNotificationsWhenNotExcludedAndMeetsCriteria`, what we need instead is an equivalent of `shouldIncludeContactDetailsInLtftInAppNotification`